plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
if you run `oc apply` or `gofabric8 deploy` or whatever with, say, 7 DCs then they all get created at the same time yet all share the same network bandwidth to download the images; so its pretty much guaranteed that some of those DCs will fail in 10 minutes from the timeouts. i.e. the more DCs you create a the same time, the longer its going to take to download all those images and the more likely a DC will just refuse to deploy.

I see overwriting the timeout value for development / CI / CD a developer thing. We want YAMLs to just work for users whether they use `oc apply`, run via the openshfit console or via `gofabric8` - as its pretty hard for end users to even know about this timeout never mind manually swizzle the templates/yamls to tweak these values. Whereas setting 1 mvn property for development / CI / CD is trivial for developers to do